src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,27 +9,329 @@ service: CloudCannon
+author: Events Coordinator
+date:
+---
+New event!
+
+Time:
+Date:
+Location:
+```
+
+You can then add this to your CloudCannon config file under the same "Posts" collection so that when creating a new content file in your Site Dashboard, you can choose between a regular blog post or an event.
+
+You can even add an `announcement` schema to multiple CloudCannon collections, though you are responsible for ensuring that each schema file will match a corresponding Astro content collection's `schema` property:
+
+```yml title="cloudcannon.config.yml"
+collections_config:
+posts:
+  path: content/blog
+  name: Posts
+  icon: post_add
+  schemas:
+    default:
+      path: .cloudcannon/schemas/post.md
+      name: Blog Post Entry
+    events:
+      path: .cloudcannon/schemas/events.md
+      name: Upcoming Event
+    announcements:
+      path: .cloudcannon/schemas/announcement.md
+      name: Important Announcement
+staff:
+  path: content/blog
+  name: Staff
+  icon: people
+  schemas:
+    default:
+      path: .cloudcannon/schemas/announcement.md
+      name: New Staff Announcement
+```
+
+## Creating a new entry
+
+In your CloudCannon Site Dashboard, you can create new content using the "Add" button. You will be able to select a an entry type from the schemas you have defined in `cloudcannon.config.yml` depending on which collection you are currently in.
+
+You can also upload files to CloudCannon, or create new files directly in your Astro project. When you save your Site changes, new files created in either location will be synchronized and available in both CloudCannon and your Astro project.
+
+The following example will create a new blog post from the CloudCannon Site Dashboard "Posts" collection using the `post.md` template schema created to satisfy the `blog` Astro content collection:
+
+<Steps>
+1. In the CloudCannon Site Dashbaord, navigate to the collection representing the kind of content you want to add. For example, navigate to the "Posts" collection to add a new blog post.
+
+2. Use the "Add" button to create a new post. If you have configured CloudCannon `post.md` schema, then you can choose the default blog post entry to create a new post.
+
+3. Fill the necessary fields in the sidebar of your Content Editor (e.g. `title`, `author`, `date`), and post content and save your post.
+
+3. This post is saved locally in CloudCannon and should now be visible from your Site Dashboard in your "Posts" collection. You can view and edit all your individual posts from this dashboard page.
+
+4. When you are ready to commit this new post back to your Astro repository, select save in the Site navigation sidebar from your Site Dashboard. This will show you all unsaved changes made to your site since your last commit back to your repository and allow you to review and select which ones to save or discard.
+
+5. Return to view your Astro project files. You will now find a new `.md` file inside the specified directory for this new post, for example:
```

Comment:
I'm guessing we should mention that you should pull new changes from git at this point?